src/data/kaws.graphql

```diff
@@ -141,6 +141,7 @@ enum GroupTypes {
 
 type Query {
   collections(
+    slugs: [String!]
```

Comment:
This was added as part of [this KAWS](https://github.com/artsy/kaws/pull/225) PR.